iosHyperskillApp/iosHyperskillApp/Sources/Modules/ProgressScreen/Views/Project/ProgressScreenProjectProgressContentView.swift

```diff
@@ -73,6 +75,13 @@ struct ProgressScreenProjectProgressContentView: View {
                     subtitle: Strings.ProgressScreen.Project.stages
                 )
             }
+
+            Button(Strings.ProgressScreen.Project.changeProject) {
+                onChangeProjectTap()
+            }
+            .foregroundColor(Color(ColorPalette.primary))
+            .font(.subheadline)
+            .frame(maxWidth: .infinity)
```

Comment:
What is the purpose of `maxWidth: .infinity` here?

If you want to increase the touchable area then you should make a button at least 44 height and apply `contentShape` modifier